src/sections/aliases.tsx

```diff
@@ -0,0 +1,344 @@
+import {
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from '@mui/material';
+import Section from '../future-hopr-lib-components/Section';
+import { useAppDispatch, useAppSelector } from '../store';
+import { useEffect, useRef, useState } from 'react';
+import { actionsAsync } from '../store/slices/node/actionsAsync';
+import { exportToCsv } from '../utils/helpers';
+import { utils } from '@hoprnet/hopr-sdk';
+const { APIError } = utils;
+
+function AliasesPage() {
+  const dispatch = useAppDispatch();
+  const aliases = useAppSelector((selector) => selector.sdk.aliases);
+  const loginData = useAppSelector((selector) => selector.auth.loginData);
+  const [errors, set_errors] = useState<
+    { status: string | undefined; error: string | undefined; alias: string }[]
+  >([]);
+
+  useEffect(() => {
+    if (loginData.apiEndpoint && loginData.apiToken) {
+      dispatch(
+        actionsAsync.getAliasesThunk({
+          apiEndpoint: loginData.apiEndpoint,
+          apiToken: loginData.apiToken,
+        })
+      );
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [loginData]);
+
+  return (
+    <Section className="Section--aliases" id="Section--aliases" yellow>
+      <h2>Add new alias</h2>
+      <CreateAliasForm />
+      <h2>Aliases table</h2>
```

Comment:
Under the table please add the same error handling as you added in the Add new alias